src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,129 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/**
+ * this class retrieves limelight values from the networktable
+ */
+public class Limelight extends SubsystemBase {
```

Comment:
At some point we should rename this to refer to a specific limelight (since we'll have two) and then we can create another limelight class for the other limelight.